UEBPrep/app/controllers/homepage_controller.rb

```diff
@@ -1,4 +1,25 @@
 class HomepageController < ApplicationController
-  def index
```

Comment:
let's not delete this default method, it's a default controller action that should just redirect back to the homepage, useful if elsewhere in the app we won't to redirect to the homepage.